pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
```

Comment:
Also, by returning the text we're returning (from example of production instance right now) "48 unclassified". This will resolve as True when treated as a boolean, but does not return an integer, and will likely never be "0". I think is would be "0 unclassified" at best.